### PR TITLE
[WIP] Windows support (MinGW) alt version

### PR DESCRIPTION
**Pull request by riddell-stan**
- [ ] standardize temporary_dir / temporary_directory
- [ ] stdout/stderr redirect as separate PR?
- [ ] call cleanup() on Linux/macOS
- [x] move the stderr functions outside the (old) context manager

### CODE REVIEW

File: httpstan/models.py [hunk 1]
```diff
@@ -193,119 +193,119 @@ def _build_extension_module(
...
-
-        # silence stdout and stderr for compilation, if stderr is silenceable
-        # silence stdout too as cythonizing prints a couple of lines to stdout
-        stream = tempfile.TemporaryFile(prefix="httpstan_")
-        redirect_stderr = _has_fileno(sys.stderr)
-        compiler_output = ""
-        if redirect_stderr:
-            orig_stdout = _redirect_stdout()
-            orig_stderr = _redirect_stderr_to(stream)
+    temporary_dir = tempfile.TemporaryDirectory().name:
```
Comment (ahartikainen) [inline httpstan/models.py]: There is an extra `:` at the end of the line

Comment (riddell-stan) [reply httpstan/models.py]: Thanks. Fixed.

**Comment by riddell-stan** [comment]
If this doesn't work (file not found error), restart afresh with the changes in #151 that do appear to work.

**Comment by riddell-stan** [comment]
Important pieces of #151 have been added or are part of other PRs.

**Pull request closed without merging**